specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -1063,6 +1063,268 @@
           "readOnly": true
         }
       }
+    },
+    "ContainerAppsJobContainer": {
+      "description": "Container App container definition",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ContainerAppsJobBaseContainer"
+        }
+      ],
+      "properties": {
+        "probes": {
+          "description": "List of probes for the container.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerAppsJobProbe"
+          },
+          "x-ms-identifiers": [
+            "type"
+          ]
+        }
+      }
+    },
+    "ContainerAppsJobProbe": {
```

Comment:
@vinisoto, I added ContainerAppsJobProbe since I see that in ContainerJobsProbe spec we have an additional property - "grpc" which is not there in [ContainerAppProbe](https://github.com/microsoft/k4apps/blob/e6580bb9c63de0f5a23e647bc7830d48ad25dac4/charts/k8se/templates/crds/k8se.microsoft.com_containerappjobs.yaml#L234). If we don't need it, we can reuse.

Reply:
@vinisoto, resolving this comment, since we are using probes defined for ContainerApps for now as grpc is not decided for V1.